package/piwebcam/camera.txt

```diff
@@ -3,5 +3,9 @@
 # Use v4l2-ctl -L or have a look at camera.txt.example
 # to view all possible settings
 
-auto_exposure_bias=3
+auto_exposure=0
```

Comment:
` auto_exposure=0` seems to be the default. Should we drop it?

Reply:
Changed it with newest push!